internal/services/aws/kms_test.go

```
@@ -0,0 +1,115 @@
+	id := awssdk.ToString(in.KeyId)
+	manager := kmstypes.KeyManagerTypeCustomer
+	keySpec := kmstypes.KeySpecSymmetricDefault
+	keyState := kmstypes.KeyStateEnabled
+	if id == "b" {
+		keySpec = kmstypes.KeySpecRsa2048
+	} else if id == "c" {
+		keyState = kmstypes.KeyStateDisabled
+	}
+	return &kms.DescribeKeyOutput{KeyMetadata: &kmstypes.KeyMetadata{KeyId: in.KeyId, Arn: awssdk.String("arn:" + id), KeyState: keyState, KeyManager: manager, KeySpec: keySpec, Origin: kmstypes.OriginTypeAwsKms}}, nil
+}
+func (mockKMSClient) ListAliases(context.Context, *kms.ListAliasesInput, ...func(*kms.Options)) (*kms.ListAliasesOutput, error) {
```

golangci-lint (2.12.2)

[error] copylocks: ListAliases passes lock by value: unic/internal/services/aws.mockKMSClient contains sync.Mutex

(govet)